testing/database_tests.js

```diff
@@ -1,25 +1,77 @@
 const test = require('tape');
+const pgp = require('pg-promise')();
 const { db } = require('./../server/database/db_connection');
+
 const { emptyTables, dummyData } = require('./../server/database/db_build');
+const {
+  getMealCard, getMealInfo, getCookInfo, getCookReviews,
+} = require('../server/database/getQueries');
+
+const { postCreateUser } = require('../server/database/postQueries');
+
+const {
+  mealCardResult, mealInfoResult, cookInfoResult, cookReviewsResult, userDetails,
+} = require('./testDummyData');
+
+let counter = 0;
+const disconnectPG = () => {
```

Comment:
Please can we chat through what this is doing this morning? Not sure and would love to understand!

Reply:
You know when we run a test or a db build. The tests finish then the terminal just waits around for some reason before ending. It turns out that it is because the db connection is still open. So right before the test ends I close the db connection to make the tests snappy. Saved all those seconds! Will show you the detail when you are next to me.